bot/seasons/valentines/savethedate.py

```diff
@@ -23,6 +23,9 @@ def __init__(self, bot):
 
     @commands.command()
     async def savethedate(self, ctx):
+        """
+        This command gives you ideas for a date with your valentine.
```

Comment:
Docstrings should be written in something called imperative mood. that means, in this case, you should write "Gives you ideas for a date with your valentine", instead of the unneccessary prefix "This command...". 

```suggestion
        Gives you ideas for what to do on a date with your valentine.
```

Reply:
got it